api/upgradeseries/upgradeseries.go

```diff
@@ -144,3 +144,19 @@ func (s *Client) StartUnitCompletion() error {
 	}
 	return results.Results[0].Error
 }
+
+// FinishUpgradeSeries starts the complete phase for all subordinate units.
+func (s *Client) FinishUpgradeSeries() error {
```

Comment:
Maybe "CompleteUpgradeSeries" so all the verbiage is congruent?

Reply:
I chose finish upgrade series because the word complete is getting somewhat overused and confusing.